engines/experiment/examples/plugins/hardcoded/runner_test.go

```diff
@@ -0,0 +1,94 @@
+package hardcoded
+
+import (
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/gojek/turing/engines/experiment/manager"
+	"github.com/gojek/turing/engines/experiment/pkg/request"
+	"github.com/gojek/turing/engines/experiment/runner"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExperimentRunner_GetTreatmentForRequest(t *testing.T) {
```

Comment:
Thanks for adding this test as an initial check to see if it's on the sample experiment's engine works :)